src/main/java/net/pms/dlna/RealFile.java

```diff
@@ -244,18 +250,23 @@ public synchronized void resolve() {
 					if (connection != null) {
 						connection.setAutoCommit(false);
 						try {
-							MediaInfo media = MediaTableFiles.getData(connection, fileName, file.lastModified());
+							MediaInfo media = MediaTableFiles.getMediaInfo(connection, fileName, file.lastModified());
 
 							setExternalSubtitlesParsed();
 							if (media != null) {
 								setMedia(media);
 								if (configuration.isDisableSubtitles() && getMedia().isVideo()) {
 									// clean subtitles obtained from the database when they are disabled but keep them in the database for the future use
+									// FIXME this hack should not be done
```

Comment:
Is there a plan to fix this?

Reply:
Yes.
This should not be done as a renderer can override the "disable subs", and then all renderer will be without subs.